### ISSUE TITLE
Problem building IOS

### ISSUE THREAD

**Issue by nebnhoj** [flutter/flutter#69671]
I'm having a problem running legacy application in IOS.
<!-- Thank you for using Flutter!

     If you are looking for support, please check out our documentation
     or consider asking a question on Stack Overflow:
      * https://flutter.dev/
      * https://api.flutter.dev/
      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent

     If you have found a bug or if our documentation doesn't have an answer
     to what you're looking for, then fill out the template below. Please read
     our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
-->

## Steps to Reproduce

<!-- You must include full steps to reproduce so that we can reproduce the problem. -->

1. Run `flutter run ios`.
2. It takes 34s to build then it always fail


**Expected results:** <!-- what did you want to see? -->

**Actual results:** <!-- what did you see? -->

<details>
  <summary>Logs</summary>

<!--
      Run your application with `flutter run --verbose` and attach all the
      log output below between the lines with the backticks. If there is an
      exception, please see if the error message includes enough information
      to explain how to solve the issue.
-->

```
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+12/ios/Classes/FLTImagePickerPlugin.m:149:20: warning: 'UIAlertView' is deprecated: first
    deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead [-Wdeprecated-declarations]
        [[[UIAlertView alloc] initWithTitle:@"Error"
                       ^
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/image_picker/image_picker-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIAlertView.h:27:12:
    note: 'UIAlertView' has been explicitly marked deprecated here
    @interface UIAlertView : UIView
               ^
    1 warning generated.
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+12/ios/Classes/FLTImagePickerPhotoAssetUtil.m:17:44: warning:
    'UIImagePickerControllerReferenceURL' is deprecated: first deprecated in iOS 11.0 - Will be removed in a future release, use PHPicker. [-Wdeprecated-declarations]
      NSURL *referenceURL = [info objectForKey:UIImagePickerControllerReferenceURL];
                                               ^
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/image_picker/image_picker-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIImagePickerControl
    ler.h:62:51: note: 'UIImagePickerControllerReferenceURL' has been explicitly marked deprecated here
    UIKIT_EXTERN UIImagePickerControllerInfoKey const UIImagePickerControllerReferenceURL API_DEPRECATED("Will be removed in a future release, use PHPicker.", ios(4.1, 11.0))
    API_UNAVAILABLE(tvos); // an NSURL that references an asset in the AssetsLibrary framework
                                                      ^
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+12/ios/Classes/FLTImagePickerPhotoAssetUtil.m:21:47: warning:
    'fetchAssetsWithALAssetURLs:options:' is deprecated: first deprecated in iOS 11 - Will be removed in a future release [-Wdeprecated-declarations]
      PHFetchResult<PHAsset *> *result = [PHAsset fetchAssetsWithALAssetURLs:@[ referenceURL ]
                                                  ^
    In module 'Photos' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+12/ios/Classes/FLTImagePickerPhotoAssetUtil.h:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Photos.framework/Headers/PHAsset.h:74:1:
    note: 'fetchAssetsWithALAssetURLs:options:' has been explicitly marked deprecated here
    + (PHFetchResult<PHAsset *> *)fetchAssetsWithALAssetURLs:(NSArray<NSURL *> *)assetURLs options:(nullable PHFetchOptions *)options API_DEPRECATED("Will be removed in a future release",
    ios(8, 11), tvos(8, 11)) API_UNAVAILABLE(macos);
    ^
    2 warnings generated.
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:61:1: warning: implementing deprecated method
    [-Wdeprecated-implementations]
    - (void)captureOutput:(AVCapturePhotoOutput *)output
    ^
    In module 'AVFoundation' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhot
    oOutput.h:682:1: note: method 'captureOutput:didFinishProcessingPhotoSampleBuffer:previewPhotoSampleBuffer:resolvedSettings:bracketSettings:error:' declared here
    - (void)captureOutput:(AVCapturePhotoOutput *)output didFinishProcessingPhotoSampleBuffer:(nullable CMSampleBufferRef)photoSampleBuffer previewPhotoSampleBuffer:(nullable
    CMSampleBufferRef)previewPhotoSampleBuffer resolvedSettings:(AVCaptureResolvedPhotoSettings *)resolvedSettings bracketSettings:(nullable AVCaptureBracketedStillImageSettings
    *)bracketSettings error:(nullable NSError *)error API_DEPRECATED_WITH_REPLACEMENT("-captureOutput:didFinishProcessingPhoto:error:", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:73:7: warning:
    'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' is deprecated: first deprecated in iOS 11.0 [-Wdeprecated-declarations]
          JPEGPhotoDataRepresentationForJPEGSampleBuffer:photoSampleBuffer
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          -[AVCapturePhoto fileDataRepresentation]
    In module 'AVFoundation' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhot
    oOutput.h:492:1: note: 'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' has been explicitly marked deprecated here
    + (nullable NSData *)JPEGPhotoDataRepresentationForJPEGSampleBuffer:(CMSampleBufferRef)JPEGSampleBuffer previewPhotoSampleBuffer:(nullable CMSampleBufferRef)previewPhotoSampleBuffer
    API_DEPRECATED_WITH_REPLACEMENT("-[AVCapturePhoto fileDataRepresentation]", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:339:13: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first
    deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
        while (!OSAtomicCompareAndSwapPtrBarrier(old, newBuffer, (void **)&_latestPixelBuffer)) {
                ^
    In module 'Darwin' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note:
    'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:561:11: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first
    deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
      while (!OSAtomicCompareAndSwapPtrBarrier(pixelBuffer, nil, (void **)&_latestPixelBuffer)) {
              ^
    In module 'Darwin' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note:
    'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:679:36: warning: 'AVVideoCodecH264' is deprecated: first deprecated in
    iOS 11.0 [-Wdeprecated-declarations]
          dictionaryWithObjectsAndKeys:AVVideoCodecH264, AVVideoCodecKey,
                                       ^~~~~~~~~~~~~~~~
                                       AVVideoCodecTypeH264
    In module 'AVFoundation' imported from /Users/macbookpro/FlutterSDK/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+11/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVVideoSettin
    gs.h:57:29: note: 'AVVideoCodecH264' has been explicitly marked deprecated here
            AVF_EXPORT NSString *const AVVideoCodecH264 /* @"avc1" */                               API_DEPRECATED_WITH_REPLACEMENT("AVVideoCodecTypeH264", macos(10.7, 10.13), ios(4.0, 11.0),
            tvos(9.0, 11.0)) API_UNAVAILABLE(watchos);
                                       ^
    5 warnings generated.
    /Users/macbookpro/regna_app/ios/Pods/FBSDKCoreKit/FBSDKCoreKit/FBSDKCoreKit/Internal/WebDialog/FBSDKWebDialogView.m:186:40: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
        [[UIApplication sharedApplication] openURL:URL];
                                           ^~~~~~~
                                           openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/FBSDKCoreKit/FBSDKCoreKit-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    1 warning generated.
    /Users/macbookpro/regna_app/ios/Pods/FBSDKCoreKit/FBSDKCoreKit/FBSDKCoreKit/Internal/BridgeAPI/ProtocolVersions/FBSDKBridgeAPIProtocolNativeV1.m:313:45: warning: 'UIPasteboardNameFind' is
    deprecated: first deprecated in iOS 10.0 - The Find pasteboard is no longer available. [-Wdeprecated-declarations]
                [pasteboardName isEqualToString:UIPasteboardNameFind]) {
                                                ^
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/FBSDKCoreKit/FBSDKCoreKit-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIPasteboard.h:17:30
    : note: 'UIPasteboardNameFind' has been explicitly marked deprecated here
    UIKIT_EXTERN NSString *const UIPasteboardNameFind API_UNAVAILABLE(tvos) API_UNAVAILABLE(watchos) API_DEPRECATED("The Find pasteboard is no longer available.", ios(3.0, 10.0));
                                 ^
    1 warning generated.
    /Users/macbookpro/regna_app/ios/Pods/FBSDKCoreKit/FBSDKCoreKit/FBSDKCoreKit/Internal/BridgeAPI/FBSDKBridgeAPI.m:200:56: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
          BOOL opened = [[UIApplication sharedApplication] openURL:url];
                                                           ^~~~~~~
                                                           openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/FBSDKCoreKit/FBSDKCoreKit-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    1 warning generated.
    /Users/macbookpro/regna_app/ios/Pods/FBSDKCoreKit/FBSDKCoreKit/FBSDKCoreKit/AppLink/FBSDKAppLinkNavigation.m:123:55: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
            } else if ([[UIApplication sharedApplication] openURL:appLinkAppURL]) {
                                                          ^~~~~~~
                                                          openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/FBSDKCoreKit/FBSDKCoreKit-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    /Users/macbookpro/regna_app/ios/Pods/FBSDKCoreKit/FBSDKCoreKit/FBSDKCoreKit/AppLink/FBSDKAppLinkNavigation.m:138:55: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
            } else if ([[UIApplication sharedApplication] openURL:appLinkBrowserURL]) {
                                                          ^~~~~~~
                                                          openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/FBSDKCoreKit/FBSDKCoreKit-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    2 warnings generated.
    /Users/macbookpro/regna_app/ios/Pods/AppAuth/Source/AppAuth/iOS/OIDExternalUserAgentIOSCustomBrowser.m:148:42: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
          [[UIApplication sharedApplication] openURL:_appStoreURL];
                                             ^~~~~~~
                                             openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/AppAuth/AppAuth-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    /Users/macbookpro/regna_app/ios/Pods/AppAuth/Source/AppAuth/iOS/OIDExternalUserAgentIOSCustomBrowser.m:156:61: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
      BOOL openedInBrowser = [[UIApplication sharedApplication] openURL:requestURL];
                                                                ^~~~~~~
                                                                openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/AppAuth/AppAuth-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    2 warnings generated.
    /Users/macbookpro/regna_app/ios/Pods/AppAuth/Source/AppAuth/iOS/OIDExternalUserAgentIOS.m:168:58: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0
    [-Wdeprecated-declarations]
        openedUserAgent = [[UIApplication sharedApplication] openURL:requestURL];
                                                             ^~~~~~~
                                                             openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/AppAuth/AppAuth-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:
    1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    1 warning generated.
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:155:38: warning: values of type 'NSUInteger' should not be used as format arguments; add an explicit cast to
    'unsigned long' instead [-Wformat]
          NSLog(@ "[features count]:%d", [features count]);
                                    ~~   ^~~~~~~~~~~~~~~~
                                    %lu  (unsigned long)
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:121:24: warning: unused variable 'detectionString' [-Wunused-variable]
          NSMutableString *detectionString = [[NSMutableString alloc] init];
                           ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:310:15: warning: 'requiredBillingAddressFields' is deprecated: first deprecated in iOS 11.0
    [-Wdeprecated-declarations]
          request.requiredBillingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                  requiredBillingContactFields
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :138:46: note: 'requiredBillingAddressFields' has been explicitly marked deprecated here
    @property (nonatomic, assign) PKAddressField requiredBillingAddressFields API_DEPRECATED_WITH_REPLACEMENT("requiredBillingContactFields", ios(8.0, 11.0), watchos(2.0, 4.0));
                                                 ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:310:46: warning: 'PKAddressFieldPostalAddress' is deprecated: first deprecated in iOS 11.0 - Use PKContactField
    and -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredBillingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                 ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:310:76: warning: 'PKAddressFieldEmail' is deprecated: first deprecated in iOS 11.0 - Use PKContactField and
    -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredBillingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                                               ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:310:98: warning: 'PKAddressFieldName' is deprecated: first deprecated in iOS 11.0 - Use PKContactField and
    -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredBillingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                                                                     ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:311:15: warning: 'requiredShippingAddressFields' is deprecated: first deprecated in iOS 11.0
    [-Wdeprecated-declarations]
          request.requiredShippingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                  requiredShippingContactFields
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :149:46: note: 'requiredShippingAddressFields' has been explicitly marked deprecated here
    @property (nonatomic, assign) PKAddressField requiredShippingAddressFields API_DEPRECATED_WITH_REPLACEMENT("requiredShippingContactFields", ios(8.0, 11.0), watchos(2.0, 4.0));
                                                 ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:311:47: warning: 'PKAddressFieldPostalAddress' is deprecated: first deprecated in iOS 11.0 - Use PKContactField
    and -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredShippingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                  ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:311:77: warning: 'PKAddressFieldEmail' is deprecated: first deprecated in iOS 11.0 - Use PKContactField and
    -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredShippingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                                                ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:311:99: warning: 'PKAddressFieldName' is deprecated: first deprecated in iOS 11.0 - Use PKContactField and
    -requiredShippingContactFields / -requiredBillingContactFields [-Wdeprecated-declarations]
          request.requiredShippingAddressFields = PKAddressFieldPostalAddress | PKAddressFieldEmail | PKAddressFieldName;
                                                                                                      ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentRequest.h
    :30:32: note: 'PKAddressField' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, PKAddressField) {
                                   ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:379:29: warning: assigning retained object to unsafe_unretained variable; object will be released after assignment
    [-Warc-unsafe-retained-assign]
          _navigationController = [
                                ^ ~
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:390:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
            [_hostViewController dismissViewControllerAnimated: TRUE completion: nil];
             ^
             self->
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:394:5: warning: implementing deprecated method [-Wdeprecated-implementations]
      } -
        ^
    In module 'PassKit' imported from /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.h:4:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/PassKit.framework/Headers/PKPaymentAuthoriza
    tionViewControllerDelegate.h:103:1: note: method 'paymentAuthorizationViewController:didAuthorizePayment:completion:' declared here
    - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController *)controller
    ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:411:24: warning: direct comparison of a string literal has undefined behavior [-Wobjc-string-compare]
        if (publishableKey != @ "") {
                           ^  ~~~~
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:411:24: note: use 'isEqual:' instead
        if (publishableKey != @ "") {
                           ^~
            ![              isEqual: ]
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:434:230: warning: 'base64Encoding' is deprecated: first deprecated in iOS 7.0 - Use
    base64EncodedStringWithOptions: instead [-Wdeprecated-declarations]
                    _result([NSString stringWithFormat: @ "{\"statusCode\":\"%@\",\"paymentData\":\"%@\",\"transactionIdentifier\":\"%@\",\"email\":\"%@\"}", asyncSuccessful == true ? @
                    "true" : @ "false", [payment.token.paymentData base64Encoding], transactionIdentifier, payment.shippingContact.emailAddress]);
                                                                                                                                                                                                                                         ^
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/tozi_plugin/tozi_plugin-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSData.h:185:1:
    note: 'base64Encoding' has been explicitly marked deprecated here
    - (NSString *)base64Encoding API_DEPRECATED("Use base64EncodedStringWithOptions: instead", macos(10.6,10.9), ios(4.0,7.0), watchos(2.0,2.0), tvos(9.0,9.0));
    ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:449:228: warning: 'base64Encoding' is deprecated: first deprecated in iOS 7.0 - Use
    base64EncodedStringWithOptions: instead [-Wdeprecated-declarations]
                  _result([NSString stringWithFormat: @ "{\"statusCode\":\"%@\",\"paymentData\":\"%@\",\"transactionIdentifier\":\"%@\",\"email\":\"%@\"}", asyncSuccessful == true ? @ "true"
                  : @ "false", [payment.token.paymentData base64Encoding], transactionIdentifier, payment.shippingContact.emailAddress]);
                                                                                                                                                                                                                                       ^
    In module 'UIKit' imported from /Users/macbookpro/regna_app/ios/Pods/Target Support Files/tozi_plugin/tozi_plugin-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSData.h:185:1:
    note: 'base64Encoding' has been explicitly marked deprecated here
    - (NSString *)base64Encoding API_DEPRECATED("Use base64EncodedStringWithOptions: instead", macos(10.6,10.9), ios(4.0,7.0), watchos(2.0,2.0), tvos(9.0,9.0));
    ^
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:434:17: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
                    _result([NSString stringWithFormat: @ "{\"statusCode\":\"%@\",\"paymentData\":\"%@\",\"transactionIdentifier\":\"%@\",\"email\":\"%@\"}", asyncSuccessful == true ? @
                    "true" : @ "false", [payment.token.paymentData base64Encoding], transactionIdentifier, payment.shippingContact.emailAddress]);
                    ^
                    self->
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:449:15: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
                  _result([NSString stringWithFormat: @ "{\"statusCode\":\"%@\",\"paymentData\":\"%@\",\"transactionIdentifier\":\"%@\",\"email\":\"%@\"}", asyncSuccessful == true ? @ "true"
                  : @ "false", [payment.token.paymentData base64Encoding], transactionIdentifier, payment.shippingContact.emailAddress]);
                  ^
                  self->
    /Users/macbookpro/regna_app/plugins/tozi_plugin/ios/Classes/ToziPlugin.m:557:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
            _result([NSString stringWithFormat: @ "{\"statusCode\":\"%@\",\"transactionIdentifier\":\"%@\"}", asyncSuccessful == true ? @ "true" : @ "false", transactionIdentifier]);
            ^
            self->
    19 warnings generated.
    23 warnings generated.

    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: None of the architectures in ARCHS (x86_64) are valid. Consider setting ARCHS to $(ARCHS_STANDARD) or updating it to include at least one value from VALID_ARCHS (arm64, arm64e,
    armv7s). (in target 'Runner' from project 'Runner')

Could not build the application for the simulator.
Error launching application on iPhone 8.

```

<!--
     Run `flutter analyze` and attach any output of that command below.
     If there are any analysis errors, try resolving them before filing this issue.
-->

```
```

<!-- Finally, paste the output of running `flutter doctor -v` here. -->

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.2, on Mac OS X 10.15.7 19H2, locale en-PH)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
[✓] Android Studio (version 4.0)
[✓] Connected device (1 available)

• No issues found!
MACBOOKPROs-MacBook-Pro:regna_app macbookpro$ flutter doctor -v
[✓] Flutter (Channel stable, 1.22.2, on Mac OS X 10.15.7 19H2, locale en-PH)
    • Flutter version 1.22.2 at /Users/macbookpro/FlutterSDK/flutter
    • Framework revision 84f3d28555 (3 weeks ago), 2020-10-15 16:26:19 -0700
    • Engine revision b8752bbfff
    • Dart version 2.10.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/macbookpro/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0.1, Build version 12A7300
    • CocoaPods version 1.10.0

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 50.0.1
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

 
[✓] Connected device (1 available)            
    • iPhone 8 (mobile) • E7161644-0AE3-484E-B205-BCE87CB2D158 • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-0 (simulator)

• No issues found!

```

</details>

**Comment by darshankawar** [comment]
@nebnhoj 
What is your iOS deployment target ? Also, what does legacy application mean ?
Thanks.

**Comment by nebnhoj** [comment]
> @nebnhoj
> What is your iOS deployment target ? Also, what does legacy application mean ?
> Thanks.

The current IOS deployment target is IOS 11, legacy application is an outdated app of my client @darshankawar .
Thank you for your reply

**Comment by darshankawar** [comment]
@nebnhoj 
Can you try [this](https://github.com/flutter/flutter/issues/56567#issuecomment-626853824) and see if it works ?

**Comment by nebnhoj** [comment]
@darshankawar I will try this. Thank you.

**Comment by nebnhoj** [comment]
@darshankawar I can't find ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings in project file.

**Comment by darshankawar** [comment]
@nebnhoj 
Since I don't know what are your project configurations and settings, hard to tell why you don't see the file.
Please provide `flutter run ios --verbose`, `cocoapods` version.

Can you also try creating a fresh project and run the same command you were running to see if it works ? If so, then try to compare the configuration of your new project and legacy to see what's missing.

**Comment by nebnhoj** [comment]
@darshankawar 
the `flutter run --verbose` log is included in the post 
1.10.0

**Comment by darshankawar** [comment]
@nebnhoj 
Can you also try creating a fresh project and run the same command you were running to see if it works ? If so, then try to compare the configuration of your new project and legacy to see what's missing.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by tudor07** [comment]
@pedromassango please re-open this issue, it still happens

**Comment by pedromassangocode** [comment]
> Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
> All system setups can be slightly different, so it's always better to open new issues and reference related issues.
